src/DynamoCore/Models/NodeModel.cs

```diff
@@ -454,16 +463,6 @@ public bool IsUpdated
             set
             {
                 isUpdated = value;
-                if (isUpdated)
```

Comment:
Nice. There should be a law about property setters not setting other properties.